src/components/ui/Modal/ModalHeader.tsx

```diff
@@ -36,12 +36,13 @@ export const ModalHeader: FC<ModalHeaderProps> = ({
       </Tag>
 
       {(displayClose && context?.dismissible) && (
-        <IconButton
-          label="Close"
-          icon={<Remove />}
-          className="ch-close-button"
-          onClick={handleClick}
-        />
+        <span className="ch-close-button">
```

Comment:
It is. Now that the `IconButton` is wrapped in a span that provides a new positioning context, we need to preserve the outer positioning context so that we can position the close button in the location that we want to for the `ModalHeader`. We can't set these styles to the `IconButton` as it needs to be relatively positioned.

Reply:
makes sense